PortalSwift/Sources/PortalSwift/Core/Api/PortalApi.swift

```diff
@@ -0,0 +1,589 @@
+//
+//  PortalApi.swift
+//
+//  Created by Portal Labs, Inc.
+//  Copyright © 2022 Portal Labs, Inc. All rights reserved.
+//
+
+import Foundation
+
+/// The class to interface with Portal's REST API.
+public class PortalApi {
+  private var _client: ClientResponse?
```

Comment:
I'm a bit confused by the `_` prefix, it's already marked as a private variable right?

Reply:
This is because there's a public getter with the name `client` that's used by other classes.